lib/wallet/src/Cardano/Wallet/DB/Sqlite/Schema.hs

```diff
@@ -491,4 +491,22 @@ CBOR
     Primary cborTxId
     deriving Show Generic Eq
 
+Submissions
+    submissionTxId TxId  sql=tx_id
+    submissionTx W.SealedTx sql=tx
+    submissionExpiration SlotNo sql=expiration
+    submissionAcceptance (Maybe SlotNo) sql=acceptance
+    submissionWallet W.WalletId sql=wallet_id
+    submissionStatus Int sql=status
```

Comment:
```suggestion
    submissionStatus TxSubmissionStatus sql=status
```
 
I understand that we can't use the `TxStatus` from `Submissions` here, as that type is not a plain enumeration, but I nonetheless favor an enumeration type that encodes the constructor name, e.g.

```
data TxSubmissionStatus = InSubmission | InLedger | Expired
```

Avoiding magic integer constants is worth the trouble. 🤓